packages/finer-wallet/src/lib/finer-wallet.ts

```diff
@@ -49,7 +50,7 @@ export function setupFinerWallet({
         description: null,
         iconUrl,
         deprecated,
-        available: true,
+        available: isMobile(),
```

Comment:
This is not quite right. `available` means if a wallet is installed and ready to be used with. For example injected wallets will have `available` false if the extension is not installed for that particular wallet.

`isMobile()` should be checked above and will `return null` if is not mobile.